### PR TITLE
Feedback Changes

### PR DESCRIPTION
**Pull request by dinesharjani**
Changes based on a collection of feedback.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ const version = require('./package.json').version.split('.')[0];
...
-const services = require(pathForJson('services_uuid'));
-const characteristics = require(pathForJson('characteristics_uuid'));
-const descriptors = require(pathForJson('descriptors_uuid'));
+const services = require(pathForJson('service_uuids'));
+const characteristics = require(pathForJson('characteristic_uuids'));
+const descriptors = require(pathForJson('descriptor_uuids'));
 
 const companySchema = require(pathForJson('company_schema'));
-const uuidSchema = require(pathForJson('gatt_schema'));
+const attributechema = require(pathForJson('attribute_schema'));
```
Comment (bencefr) [inline index.js]: It looks like an S is missing here and in all references.

File: index.js [hunk 2]
```diff
@@ -4,12 +4,12 @@ const version = require('./package.json').version.split('.')[0];
 const pathForJson = filename => `${__dirname}/v${version}/${filename}.json`;
 
 const companies = require(pathForJson('company_ids'));
-const services = require(pathForJson('services_uuid'));
-const characteristics = require(pathForJson('characteristics_uuid'));
-const descriptors = require(pathForJson('descriptors_uuid'));
+const services = require(pathForJson('service_uuids'));
+const characteristics = require(pathForJson('characteristic_uuids'));
+const descriptors = require(pathForJson('descriptor_uuids'));
```
Comment (bencefr) [inline index.js:9]: Good catch.

Comment (dinesharjani) [reply index.js:9]: Not mine - it was Bjorn Inge's :)

File: v1/service_uuids.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    { "name": "Scan Parameters", "identifier": "org.bluetooth.service.scan_parameters" , "uuid": "1813" , "source": "gss"},
+    { "name": "Transport Discovery", "identifier": "org.bluetooth.service.transport_discovery" , "uuid": "1824" , "source": "gss"},
+    { "name": "Tx Power", "identifier": "org.bluetooth.service.tx_power" , "uuid": "1804" , "source": "gss"},
+    { "name": "User Data", "identifier": "org.bluetooth.service.user_data" , "uuid": "181C" , "source": "gss"},
+    { "name": "Weight Scale", "identifier": "org.bluetooth.service.weight_scale", "uuid": "181D" , "source": "gss"},
+
+    { "name": "Apple Notification Center Service", "identifier": "com.apple.service.notification_center", "uuid": "7905F431-B5CE-4E99-A40F-4B1E122D00D0" , "source": "apple"},
+    { "name": "Apple Media Service", "identifier": "com.apple.service.media", "uuid": "89D3502B-0F36-433A-8EF4-C502AD55F8DC" , "source": "apple"},
+
+    { "name": "micro:bit Accelerometer Service", "identifier": "com.micro.bit.service.accelerometer", "uuid": "E95D0753-251D-470A-A062-FA1922DFA9A8" , "source": "micro:bit"},
```
Comment (bihanssen) [inline v1/service_uuids.json:46]: Maybe not important, but could be good to align identifier with official domain names. For microbit that would be org.microbit.

File: v1/characteristic_uuids.json [hunk 4]
```diff
@@ -1,350 +1,350 @@
...
+    { "name": "Waist Circumference", "identifier": "org.bluetooth.characteristic.waist_circumference", "uuid": "0x2A97", "source": "gss" },
+    { "name": "Weight", "identifier": "org.bluetooth.characteristic.weight", "uuid": "0x2A98", "source": "gss" },
+    { "name": "Weight Measurement", "identifier": "org.bluetooth.characteristic.weight_measurement", "uuid": "0x2A9D", "source": "gss" },
+    { "name": "Weight Scale Feature", "identifier": "org.bluetooth.characteristic.weight_scale_feature", "uuid": "0x2A9E", "source": "gss" },
+    { "name": "Wind Chill", "identifier": "org.bluetooth.characteristic.wind_chill", "uuid": "0x2A79", "source": "gss" },
 
-    { "name": "Button State Characteristic", "identifier": "ccom.nordicsemi.blinky", "uuid": "00001524-1212-EFDE-1523-785FEABCD123" , "specification": "nordic"},
-    { "name": "LED State Characteristic", "identifier": "com.nordicsemi.blinky", "uuid": "00001525-1212-EFDE-1523-785FEABCD123" , "specification": "nordic"},
+    { "name": "Button State Characteristic", "identifier": "ccom.nordicsemi.characteristic.blinky.button_state", "uuid": "00001524-1212-EFDE-1523-785FEABCD123" , "source": "nordic"},
```
[the remark this answers is not in the document]
Comment (dinesharjani) [reply v1/characteristic_uuids.json:229]: Fixed.

File: verify.js [hunk 5]
```diff
@@ -4,20 +4,27 @@ const noDuplicates = Object.keys(db)
   .filter(key => key !== 'schemas')
   .map(list => {
     let numberOfItems;
-    let numberOfUniques;
+    let numberOfUniqueUUIDs;
     if (list === 'companies') {
       let companyCodes = db[list].map(company => company.code);
       numberOfItems = companyCodes.length;
-      numberOfUniques = (new Set(companyCodes)).size;  
+      numberOfUniqueUUIDs = (new Set(companyCodes)).size;
```
[the remark this answers is not in the document]
Comment (dinesharjani) [reply verify.js:11]: Done.

File: v1/service_uuids.json (hunk 3 above)
Comment (dinesharjani) [reply v1/service_uuids.json:46]: I agree.

**Pull request merged**